Expensify/App
# add informative text for visibility settings

### Details

Added informative text for visibiilty settings in New Room page
### Fixed Issues

$ https://github.com/Expensify/App/issues/7669

### Tests



### QA Steps

1. Click "+"
2. Click "New Room"
3. Check that there is some informative text below the Visibility settings

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/83179295/155342651-5678bb40-74ae-44c2-926a-d0e58079bf7b.mov


#### Mobile Web

https://user-images.githubusercontent.com/83179295/155342669-e066d867-64fa-4cef-b2dc-4d8003ae44a2.mov



#### Desktop


https://user-images.githubusercontent.com/83179295/155342714-bfe78e5c-1395-486e-908e-b580e90c721f.mov


#### iOS


https://user-images.githubusercontent.com/83179295/155342744-c13191f8-58e0-4275-a651-2bab6e871cac.mov



#### Android



https://user-images.githubusercontent.com/83179295/155343829-6e61faab-c191-47c8-9e17-2b9765d4fce9.mov
